README.md

```diff
@@ -11,6 +11,8 @@ Material Components for the web is the successor to [Material Design Lite](https
 
 MDC-Web strives to seamlessly incorporate into a wider range of usage contexts, from simple static websites to complex, JavaScript-heavy applications to hybrid client/server rendering systems. In short, whether you're already heavily invested in another framework or not, it should be easy to incorporate Material Components into your site in a lightweight, idiomatic fashion.
 
+> **Note:** MDC-Web is intended to be used for other developers to execute Material Design properly. MDC-Web delegates higher level capabilities to other JS frameworks and libraries. An example is that MDC-Web does not offer the latest of Web components capabilities, e.g. custom elements, Shadow DOM, HTML imports/templates. We expect other JS frameworks and libraries to deal with these configurations.
```

Comment:
I think this is a good start but I'd like to rephrase this section a bit. 

I think we could frame this better by saying that MDC-Web explicitly doesn't do things like use web components, render its own DOM, etc. so that it can be as interoperable as possible with other frameworks and libraries. It's not that we "expect other JS frameworks and libraries to deal with these configurations", but more like "we try very hard not to get in the way of what other JS frameworks and libraries do".